### PR TITLE
Init OATS->ALCS Users

### PR DESCRIPTION
**Pull request by lstod**

### CODE REVIEW

File: bin/migrate-oats-data/common/alcs_init_users.py [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+            encoding="utf-8",
+        ) as sql_file:
+            count_query = sql_file.read()
+            cursor.execute(count_query)
+            count_total = dict(cursor.fetchone())["count"]
+        logger.info(f"Total users data to insert: {count_total}")
+
+        failed_inserts = 0
+        successful_inserts_count = 0
+        last_submission_id = "-"
```
Comment (mhuseinov) [inline bin/migrate-oats-data/common/alcs_init_users.py:34]: nitpick: last_user_uuid

File: bin/migrate-oats-data/common/alcs_init_users.py [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+                    bceid_guid,
+                    email,
+                    display_name,
+                    identity_provider,
+                    preferred_username,
+                    audit_created_by
+                )
+                VALUES (
+                    %(created_guid)s,
+                    '11@11',
```
Comment (mhuseinov) [inline bin/migrate-oats-data/common/alcs_init_users.py:97]: is email a required field? If yes have you confirmed with Ansley default value?

**Comment by mhuseinov** [comment]
Could you also include import user in to both noi and application imports since those will be heavily dependent on it

File: bin/migrate-oats-data/common/alcs_init_users.py (hunk 2 above)
Comment (lstod) [reply bin/migrate-oats-data/common/alcs_init_users.py:97]: is a not null. Yet to confirm, copied obfuscation

File: bin/migrate-oats-data/common/alcs_init_users.py (hunk 1 above)
Comment (lstod) [reply bin/migrate-oats-data/common/alcs_init_users.py:34]: will change to guid

**Review by mhuseinov — APPROVED** [review]

**Comment by lstod** [comment]
> Could you also include import user in to both noi and application imports since those will be heavily dependent on it
Done. noi and application clean will not clean users

**Pull request merged**